brabbuss/overlook
# Key Functionality: login, available rooms listing

### What does this PR do?
- 1b6c60e add login modal that pops up when clicking on user profile image in header
- db76834, 3fb7533 add styling and functionality for login - will instantiate proper user or manager based on passing credentials
- 38994e3 add ability to view available rooms for booking
- 5296bd9 add toolbar for selecting by date, type, and search by user (manager view)

### How should this be tested?
1. upon page load, should be able to see available rooms when clicking 'booking' link at the top of the site
2. should be able to login using the login modal using proper user information
3. logging in as manager should take you to empty manager dashboard
4. should see toolbar, though not functional

### New Questions/Thoughts?:
1. Will need to adjust copy on links to make more sense, new issue
